jgrapht-core/src/main/java/org/jgrapht/GraphTests.java

```diff
@@ -479,6 +479,35 @@
         return new HierholzerEulerianCycle<V, E>().isEulerian(graph);
     }
 
+    /**
+     * Tests whether an undirected graph meets Ore's condition to be Hamiltonian.
+     *
+     * Let G be a (finite and simple) graph with n ≥ 3 vertices. We denote by deg v the degree of a vertex v in G, i.e. the number of incident edges in G to v.
```

Comment:
Please use math notation, see contributors guidelines in the wiki

Reply:
Done.